src/iodined.c

```diff
@@ -1719,6 +1740,17 @@ tunnel_dns(int tun_fd, int dns_fd, struct dnsfd *dns_fds, int bind_fd)
 			return 0;
 		}
 
+		/* Handle A-type query for _.***.topdomain. It happens when
```

Comment:
Can you link https://datatracker.ietf.org/doc/html/rfc7816  (qname minimisation)